Frontend/src/app/detect/prediction model/CNN.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
```

Comment:
you said, we are not gone need to run this code while executing, we decided to host in other cloud and run. Then why we need it right now.

Reply:
but it should be visible right, showing we have implemented this for detecting using model,

Reply:
ok, good